hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -166,7 +166,7 @@ const MISMATCH_TAB_COLUMNS = [
 
 const OPEN_KEY_TAB_COLUMNS = [
   {
-    title: 'Key',
+    title: 'Key Name',
```

Comment:
Why are you changing this?

Reply:
@dombizita @devmadhuu  It was suggested by Devesh

Reply:
I'm fine with it, just wanted to know the reason behind it :)

Reply:
@dombizita  Can you Merge PR